api/Objects/player.js

```diff
@@ -0,0 +1,33 @@
+export default class player{
+    constructor(id, username, score,){
+        this.id = id;
+        this.username = username;
+        this.score = score;
+        this.cards = [];
+    }
+    
+    get Cards(){
+        return this.cards;
+    }
+
+    get points(){
+        return this.score;
+    }
+
+    addCard(card){
+        this.cards.push(card);
+    }
+
+    playCard(pick, card){
```

Comment:
Can you add a comment on what this code is doing? it looks a bit confusing to me!

Reply:
Oops, that needs to be deleted.